src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -41,6 +43,60 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 });
             }
         }
+        
+        [RavenAction("/databases/*/replication/all-items", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
Looks like a debug endpoint?
Maybe `/databases/*/debug/replication/all-items` ?

Reply:
Done, explicitly not setting `IsDebugInformationEndpoint` to avoid adding this to the debug package